### PR TITLE
Add default classpath when not present

### PR DESCRIPTION
**Pull request by riteshghorse**
This PR adds a small enhancement to specifying additional dependency jars to JDBC IO. As discussed in #17216, we don't want users to specify both driver class name and the classpath when using a default driver jar. Integration tested by modifying [expand.go](https://github.com/apache/beam/blob/b0e6b561683425fe865720970ce60d45ecec11e4/sdks/go/pkg/beam/core/runtime/xlangx/expand.go#L148) to use version `2.38.0` instead of `core.SdkVersion`.
change 
`jarPath, err := expansionx.GetBeamJar(gradleTarget, core.SdkVersion)` 
to
`jarPath, err := expansionx.GetBeamJar(gradleTarget, "2.38.0")`

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Review by jrmccluskey — APPROVED** [review]
LGTM

**Comment by riteshghorse** [comment]
R: @lostluck

File: sdks/go/pkg/beam/io/xlang/jdbcio/jdbc.go
Comment (lostluck) [inline sdks/go/pkg/beam/io/xlang/jdbcio/jdbc.go:258]:
```suggestion
// With Classpath parameter:
```

**Review by lostluck — APPROVED** [review]
LGTM! One typo I'll fix (assuming committer changes are permitted on this PR).

**Pull request merged**
